ephypype/preproc.py

```diff
@@ -427,18 +430,21 @@ def generate_report(raw, ica, subj_name, basename,
     report.add_figs_to_section(fig, captions=['All IC time series'],
                                section='ICA - muscles')
 
-    psds = []
-    captions_psd = []
-    ica_src = ica.get_sources(raw)
-    for i_ic in ic_nums:
-        fig = ica_src.plot_psd(tmax=60, picks=[i_ic], fmax=140, show=False)
-        fig.set_figheight(3)
-        fig.set_figwidth(5)
-        psds.append(fig)
-        captions_psd.append('IC #' + str(i_ic))
-
-    report.add_figs_to_section(figs=psds, captions=captions_psd,
-                               section='ICA - muscles')
+    try:
+        psds = []
+        captions_psd = []
+        ica_src = ica.get_sources(raw)
+        for i_ic in ic_nums:
+            fig = ica_src.plot_psd(tmax=60, picks=[i_ic], fmax=140, show=False)
+            fig.set_figheight(3)
+            fig.set_figwidth(5)
+            psds.append(fig)
+            captions_psd.append('IC #' + str(i_ic))
+
+        report.add_figs_to_section(figs=psds, captions=captions_psd,
+                                   section='ICA - muscles')
+    except:  # noqa
```

Comment:
it's not a good idea to put `try-except` clause in your repo. It's better to do it with an `if-else` ... and raise an appropriate warning

Reply:
plot_psd raises an error. Looking at the MNE code it seems due to the fact that now to plot the PSD of 1 ICA component we have to use directly psd_multipaper. @dmalt do you have time to ckeck this? 
@jasmainak the try was a temporary fix :)